instrumentation/logback/logback-appender-1.0/library/src/main/java/io/opentelemetry/instrumentation/logback/appender/v1_0/OpenTelemetryAppender.java

```diff
@@ -25,6 +25,7 @@ public class OpenTelemetryAppender extends UnsynchronizedAppenderBase<ILoggingEv
       new LogEmitterProviderHolder();
 
   private volatile boolean captureExperimentalAttributes = false;
+  private volatile boolean captureCodeAttributes = false;
```

Comment:
I wanted first that the `captureCodeAttributes` property would be an `AtomicBoolean`, but I have noticed after that `captureExperimentalAttributes` is a `boolean`, so I keep the same way of doing